packages/core/admin/server/controllers/api-token.js

```diff
@@ -53,6 +54,26 @@ module.exports = {
     ctx.created({ data: accessToken });
   },
 
+  async refresh(ctx) {
+    const { id } = ctx.params;
+    const apiTokenService = getService('api-token');
+
+    const apiToken = await apiTokenService.getById(id);
+    if (!apiToken) {
+      ctx.notFound('API Token not found');
+      return;
+    }
+
+    if (!apiToken.lifespan) {
+      ctx.badRequest(`API Token doesn't have a lifespan to refresh`);
+      return;
+    }
+
+    const token = await apiTokenService.refresh(id);
+
+    ctx.created({ data: token });
```

Comment:
Not sure we want to send back a 201 there. maybe a 200 would be more appropriate, wdyt?

Reply:
Our tokens occupy a weird space, since they're not really JWTs but externally they work pretty similarly. But generally it's seen as "creating" an accessKey each time, so that's why I went with POST / created.

Reply:
We're not updating the accessKey in this case, though. We're just action on the expiresAt. That's why I would see it more as a generic action rather than an entity creation